packages/frontend/src/components/common/modal/DashboardCreateModal.tsx

```diff
@@ -4,20 +4,19 @@ import {
     DialogBody,
     DialogFooter,
     DialogProps,
-    FormGroup,
-    HTMLSelect,
-    InputGroup,
-    Intent,
 } from '@blueprintjs/core';
 import { Dashboard, Space } from '@lightdash/common';
 import { FC, useCallback, useState } from 'react';
+import { useForm } from 'react-hook-form';
```

Comment:
Wondering the reason for using `react-hook-form` rather than using mantine's own?

Reply:
I just felt like it was out of scope to migrate the modal to Mantine but now I realise I could've totally done it, too late 🤷🏼